packages/icons/src/mod/index.js

```diff
@@ -0,0 +1,19 @@
+import ModBike from "./mod-bike";
```

Comment:
I think the `mod/` directory (and `Mod*` icon names) should be called `trimet/` (TriMet)

Reply:
OK, never mind. I guess there already is a TriMet directory. This is confusing to me. Can you add to the package's README about different use cases and distinguish between the icon sets? Before you do that though, let's hammer out this mod vs. trimet distinction.

Reply:
There is already a trimet/ folder.